core/domain/exp_domain.py

```diff
@@ -4647,7 +4646,9 @@ def _fix_rte_tags(
         Returns:
             str. Returns the updated html value.
         """
-        soup = bs4.BeautifulSoup(html, 'html.parser')
+        soup = bs4.BeautifulSoup(
```

Comment:
Hey @kevintab95, I have also added this encoding part from the doc [here](https://docs.google.com/document/d/1HFZdyQqG47dgbakz05lzSejcd8RlNAQaPWPfjGt4sWY/edit?usp=sharing) just to avoid any incase problem. Please let me know if this sounds good.

Reply:
Are you sure this will not cause other issues? Have you tested to make sure it does not?

Reply:
Isn't this the recommended way to use the soup to avoid any invalid changes? All the tests do run successfully and it not causing any problem but let me know what do you think. If it is not necessary and recommended I will remove it.
Thanks

Reply:
Please manually test it to ensure that it does not result in any other encoding issues. It can potentially delay things even further.

Reply:
I think I should remove this part, apart from my changes we do have many types of data present and the changes seemed to work fine the previous way. Just to avoid any kind of further delay or surprises, I'm removing it.
Thanks